package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "solid-auth-extension-chrome",
-  "version": "1.0.0",
+  "name": "solid-identity-manager",
+  "version": "1.0.1",
```

Comment:
Don't bump to 1.0.1. yet. I want the first version to be 1.0.0.

Reply:
✔️